roles/aodh/tasks/main.yml

```diff
@@ -27,6 +28,17 @@
     - aodh-evaluator
     - aodh-listener
     - aodh-notifier
+  when: os == 'ubuntu'
+
+- name: install aodh services (rhel)
+  yum: pkg={{ item }} state=installed
```

Comment:
This does not seem right. openstack-distro is one that installs the packages. HEre we just create systems_service. Also, we will need to re-visit the systemd service definitions in openstack-meta.